stencils/pace/stencils/testing/grid.py

```diff
@@ -1,16 +1,16 @@
+# mypy: ignore-errors
 import dataclasses
 import functools
 from typing import Any, Iterable, Mapping, Optional, Sequence, Tuple, Union
 
 import numpy as np
 from gt4py import gtscript
 
-import fv3core.utils.global_config as global_config
 import pace.util
-from fv3core.grid import MetricTerms
 from pace.dsl import gt4py_utils as utils
-from pace.dsl.stencil import GridIndexing, StencilConfig, StencilFactory
+from pace.dsl.stencil import GridIndexing
```

Comment:
Ideally only Grid should remain in this file, the other things should be moved out of `testing`. If we no longer pass `Grid` to `axis_offsets`, you should be able to modify it to only accept GridIndexing, which will allow you to move it out of this file. If we do still pass `Grid` to `axis_offsets`, you can leave the legacy function in this file and call it from whichever testing code is using it.